src/index.html

```diff
@@ -1,17 +1,169 @@
+          <input type="date"
+            name="date-birth"
+            id="date-birth"
+            placeholder="date"
+          >
+        </div>
+        <div class="form-filed">
+          <label for="accept">I accept the term of the agreement</label>
+          <input type="checkbox"
+            name="accept-agreement"
+            id="accept"
+            value="yes"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-box">
+        <legend>Registration</legend>
+        <div class="form-filed">
+          <label for="email">E-mail: </label>
+          <input type="email"
+            placeholder="email@example.com"
+            name="email"
+            id="email"
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-filed">
+          <label for="password">Password: </label>
+          <input type="password"
+            name="password"
+            id="password"
+            minlength="5"
+            maxlength="15"
+            autocomplete="off"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-box">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-filed">
+          <label for="cats"
+            >Do you love cats?
+            <input type="radio" 
+            name="cats"  
+            value="Yes"
+            > Yes
+            <input type="radio"
+             name="cats" 
+             value="No"
+            > No
+          </label>
+        </div>
+        <div class="form-filed">
+          <label for="color">What is your favorite color?</label>
+          <input type="color" 
+          name="color" 
+          id="color"
+          >
+        </div>
+        <div class="form-filed">
+          <label for="bed-time">What time do you go to bed?</label>
+          <input type="time" 
+            name="bed-time" 
+            id="bed-time"
+          >
+        </div>
+        <div class="form-filed">
+          <label for="car-select">What are your favorite brand of cars?</label>
+          <select name="car-select" 
+          id="car"
+          multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+            <option disabled></option>
+          </select>
+        </div>
+        <div class="form-filed">
+          <label for="work"> How do you rate our work?</label>
+          <input type="range"
+            name="work"
+            id="work"
+            min="0"
+            max="10"
+            value="0"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-box">
+        <legend>Additional info:</legend>
+        <div class="form-filed">
+          <label for="comments">Comments:</label>
+          <textarea name="comments"
+            id="comments"
+            placeholder="Something"
+            autocomplete="off"
+          >
+          </textarea>
```

Comment:
Check proper textarea formating. It is a bit different
_<textarea> is a tricky tag to format:
It requires you to put a closing tag on the same line as the > of the opening tag, not to have default content, and be able to see the placeholder.
At the same time, if it has several attributes, you need to put each of them on its own line._